bin/eslint-formatters-calypso/src/lib/parsing-errors.js

```diff
@@ -0,0 +1,27 @@
+/* eslint strict: "off" */
+
+'use strict';
+
+module.exports = function( report ) {
+	// deep clone the report, so we can create a new one
+	// to tweak and edit in place - as this function remains pure.
+	const newReport = [];
```

Comment:
Are we deep-cloning the report? The comments looks like a copy-paste from the one in `lib/eslines.js`.